Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -157,7 +157,10 @@ - (void)commonInit
 #pragma GCC diagnostic ignored "-Wunused-result"
     [ExperienceUpgradeFinder sharedManager];
 #pragma GCC diagnostic pop
+}
 
+- (void)observeNotifications
```

Comment:
We don't want to observe notifications until the view has loaded, since we want to "reload the table" in response to some notifications and that's not safe to do until the mappings are created.